open-company/open-company-web
# Big web

card: https://trello.com/c/gpApBZ9G

NB: some of the style details has yet to be defined as per the [design feedbacks doc](https://docs.google.com/document/d/1c61v8mkPEjEbDDsWoChjnHRjbSHSvoHGmSQbeGObZYg/edit) so try not to be too sharp.
The type of layout is chosen checking the window width, until 970px is considered mobile, above it it's big web, for now this doesn't take into account the browser's user agent and there is no distinction between smartphones and tablets.

To test:
- check the mock for big web
- try to expand/collapse some topics
- the topic expand animation has to be improved yet: i'm waiting for some feedback on the current one
- expand a topic then click on another topic in the same row: there is no animation for this yet, i'll work on it when i have some feedback on the current expand animation and maybe some guidelines
- check that this doesn't break anything on mobile (activate the chrome device mode or check this on your device)
